package.json

```diff
@@ -13,13 +13,16 @@
   "keywords": [
     "babel"
   ],
-  "peerDependencies": {
```

Comment:
Ok, I will change `babel` to `babel-core`. And babel 6.x require the new version (2.x) of this package.